draft-ietf-core-dns-over-coap.md

```diff
@@ -391,7 +392,20 @@ A successful response:
       ;; ANSWER SECTION:
       ;example.org.         79689   IN      AAAA    2001:db8:1::1:2:3:4
 
-When a DNS error—NotImp (RCODE = 4) in response to a DNS Update (OPCODE = 5) for "example.org" in this case—is noted in the DNS response, the CoAP response still indicates success.
+When a DNS error—NxDomain (RCODE = 3) for "does.not.exist" in this case—is noted in the DNS response, the CoAP response still indicates success.
+
+    2.05 Content
+    Content-Format: application/dns-message
+    Payload (human-readable):
+    ;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 0
+    ;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0
+
+    ;; OPT PSEUDOSECTION:
+    ; EDNS: version: 0, flags:; udp: 0
+    ;; QUESTION SECTION:
+    ;does.not.exist.                IN      A
+
+As described in {{sec:content-format}}, a DoC server uses NotImp (RCODE = 4) if it does not support an OPCODE—a DNS Update (OPCODE = 5) for "example.org" in this case.
```

Comment:
It looks like an example is missing here - it mentions an "example.org" DNS Update which isn't shown. The above example now has "does.not.exist" as name.

Reply:
Or does "example.org" refer back to the first example with "example.org" query? But that wasn't a DNS Update.

Reply:
Sorry, Github was hiding the example, it's below ...